INSTALL.txt

```diff
@@ -12,7 +12,7 @@ Before installing Picard, you need to have these:
  * PyQt 4.5 or newer
    http://www.riverbankcomputing.co.uk/software/pyqt/download
 
- * Mutagen 1.22 or newer (1.23 or newer for AIFF support)
+ * Mutagen 1.37 or newer
```

Comment:
fyi, all changes ever made to mutagen are relicensed, so theoretically any previous version is now gplv2+ as well. But IANAL...